tests/test_install.ts

```diff
@@ -108,9 +112,9 @@ describe("Agama test", function () {
             if (!fs.existsSync(dir)) fs.mkdirSync(dir);
 
             // base file name for the dumps
-            const name = path.join(dir, this.currentTest!.title.replace(/[^a-zA-Z0-9]/g, "_"));
+            const name: string = path.join(dir, this.currentTest!.title.replace(/[^a-zA-Z0-9]/g, "_"));
```

Comment:
please check again all this extra types in the whole file, it should be not required with the current TS configuration and we can rely for these cases in inferred types.

Reply:
c3b4d56